i3pystatus/openstack_vms.py

```diff
@@ -0,0 +1,69 @@
+from i3pystatus import IntervalModule
+from .core.util import round_dict
+# requires python-novaclient
```

Comment:
Please put that in the module description below, too.

You also need to add `novaclient.v2` to the mocks list at https://github.com/enkore/i3pystatus/blob/master/docs/conf.py#L23 (yeah, this is a bit ... uhm... but it's necessary to allow automatic generation of the docs without actually *needing* all the modules installed.